### PR TITLE
MOSIP-15189 Validate biometric signature in packetmanager

### PR DESCRIPTION
**Pull request by neosoftSatish**

### CODE REVIEW

File: sandbox/registration-processor-mz.properties [hunk 1]
```diff
@@ -465,6 +465,10 @@ mosip.kernel.jsonvalidator.file-storage-uri=${spring.cloud.config.uri}/${spring.
 
 registration.processor.validateApplicantDocument=true
 
+# To enable/disable trust validation of a digital id signature
+mosip.regproc.packet-validator.signature.disable-trust-validation=false
+mosip.regproc.common.before-cbeff-others-attibute.reg-client-versions=1.2.0-SNAPSHOT
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties]: This is common property for backward compatibility check, so add this before " WORKFLOW ENGINE PROPERTIES " section.
Change the value to 1.1.3,1.1.4,1.1.5
Please add appropriate comment that describes the property and format of the value to be given

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-mz.properties (hunk 1 above)
Comment (neosoftSatish) [reply sandbox/registration-processor-mz.properties]: corrected as mentioned...

Comment (vishwa-vyom) [reply sandbox/registration-processor-mz.properties]: Change the comments to "List of old reg client versions with others attribute not present but currently supported for registration, this is required to take care of the backward compatibility checks"

Comment (neosoftSatish) [reply sandbox/registration-processor-mz.properties]: changed the comment.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**